src/main/resources/l10n/JabRef_en.properties

```diff
@@ -580,8 +580,6 @@ No\ journal\ names\ could\ be\ abbreviated.=No journal names could be abbreviate
 
 No\ journal\ names\ could\ be\ unabbreviated.=No journal names could be unabbreviated.
 
-No\ DOI\ data\ exists=No DOI data exists
```

Comment:
Why was this string removed? Can't it be kept?

Introducing a new string causes load on ~15 translators.

Reply:
See https://github.com/JabRef/jabref/pull/8839#discussion_r878881438

> In my opinion "No DOI data was found" sounds better :)
>
> Imagine: A user wants to fetch DOI metadata from the web; Popup was triggerd: "No DOI data exists" --> user might think DOI data does not exist, even though it indeed MIGHT exist, but JabRef just could not find it or there was some error on client side.

Reply:
Then, the change is OK for me!